contents/docs/user-guides/trends.mdx

```diff
@@ -1,58 +1,324 @@
+[Filters](#filtering-events) allow us to only plot data for a subset of events based on their properties. For example, we might only want to see `$pageview` events for a specific URL, or only see events from users within specific countries, which can all be done using filters!
+
+To add a filter to an event series, click on the <FilterIcon /> icon next to the event name, and then click the 'Add filter' button.
+This will bring up a dropdown where you can search through all event properties and choose the one you would like to filter based on. For this example, we'll choose the 'Device Type' event so that we can plot data only for Desktop users.
+
+![a filter that only shows events from desktop users](../../images/docs/user-guides/trends/basic-filter.png)
+
+Read more about filters in the [filtering events](#filtering-events) section.
+
+#### Changing the time range
+
+When you first create an insight, PostHog will default to only plotting data from the last 7 days. This is great for getting a short term snapshot or digging in deeply to a specific time period, but oftentimes you'll want to see how your metrics change over the long term.
+This can be done using the [display options at the top of the chart](#changing-the-display-options) and setting the 'Date range' to `Last 90 days`.
+
+> **Note: ** Keep in mind that if you set your date range to be very large, it may severely impact your query performance as the number of events we need to process can increase dramatically
+
+![showing a chart over a longer date range (90 days)](../../images/docs/user-guides/trends/longer-date-range.png)
+
+> ** Note: ** You may see a portion of the chart plotted as a dotted line, which simply means that data for that period is still being collected
+
+#### Saving and editing a trend
+
+Now that we have our insight set up, the last step is to give it a name and save it so we can view it later! To give this Insight a name, you can click on the pencil icon next to the title, and then click the 'Save' button.
+
+If you ever want to go back and modify a trend, you can do so by clicking on its name in the 'Insight' tab, and then clicking the 'Edit' button in the top right.
+
+Now that you've mastered the basics of the trends insight, we'd encourage you to read through following sections to get a better idea of all the features that trends have to offer.
+
+## Filtering events
+
+PostHog offers powerful ways of filtering down events based on their properties. A single filter is composed of three different parts:
+
+#### The property to filter based on
+
+**Example: _(Current URL, Browser, etc)_**
+
+This is the property that we want to filter based on. These properties can be:
+
+-   Properties on the event itself
+-   Properties on the person who sent the event
+-   Properties on Groups that this event is a member of _(See [Group analytics](/docs/user-guides/group-analytics))_
```

Comment:
We have a few more options here :) 
- Cohorts
- Autocaptured elements
- sessions

Reply:
Added these to the list :+1: